README.md

```diff
@@ -20,3 +20,19 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+## Answer 
+Let's start with the best-case runtime analysis. For the best case that would be a sorted list that was put through the algorithm. This will give you a run time $O(n)$. 
```

Comment:
Are you sure? It looks to me like you're always generating all permutations first.

Reply:
That was my mistake I thought it was going through once and not multiple times.